app/vmselect/main.go

```diff
@@ -292,6 +292,11 @@ func requestHandler(w http.ResponseWriter, r *http.Request) bool {
 		httpserver.Errorf(w, r, "auth error: %s", err)
 		return true
 	}
+	if at == nil {
+		// the only option for at to be nil is when p.AuthToken == "multitenant"
```

Comment:
I think, explicit error must be returned in this case. That vmselect doesn't support multitenant endpoint.

Reply:
Before this change it printed:
```
remoteAddr: "127.0.0.1:64861"; requestURI: /select/multitenant/prometheus; unsupported path requested: "/select/multitenant/prometheus"
```
Now it prints:
```
remoteAddr: "127.0.0.1:64515"; requestURI: /select/multitenant/prometheus; auth error: tenantID must be specified explicitly in the request path
```


I find the first better. Wdyt @f41gh7 ?